toolkit/scss/_previous-next-navigation.scss

```diff
@@ -0,0 +1,127 @@
+/*
+Taken from the GOV.UK component at
+https://github.com/alphagov/static/blob/a9d462e71709d2ff6348bcce7e8c625af2b86114/app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss
+and
+https://github.com/alphagov/static/blob/da8aeeaa749093eab30286d7fc9f965533b66f47/app/assets/stylesheets/styleguide/_conditionals2.scss
+*/
+@import "_colours.scss";
+@import "_typography.scss";
+
+// Media query helpers. These make producing IE layouts
+// super easy.
+
+// These are desktop and down media queries
+
+// There is also a local version of this in Smartanswers.
+
+$is-ie: false !default;
+
+@mixin media-down($size: false, $max-width: false, $min-width: false) {
```

Comment:
This is legacy code from govuk which will, in time, be converted to mobile-up. Could we take this opportunity to make the switch here?

If we converted the `.govuk-previous-and-next-navigation` block to be mobile-up we also wouldn't need to include this mixin as we could just import `_conditionals.scss` from the govuk frontend toolkit.